tests/test_extension.py

```diff
@@ -216,6 +216,30 @@ class Model(Base):
         assert "pydantic-field" not in package["Model.a"].labels
 
 
+def test_extra_bases() -> None:
```

Comment:
I'm not sure if it's a good test since both classes are implemented in the same package. Should we split them to separate packages? or what's the best way to test this?